.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         # use all supported versions from https://devguide.python.org/versions/
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
there is at least one occurence where it says

> # TODO: can be imported directly from `typing` with python >= 3.8

can you have a look at that?

Reply:
[2cd2b81](https://github.com/geo-engine/geoengine-python/pull/144/commits/2cd2b81d1ad6fb0db1482a485989f8b940af4dea)

Seems unrelated or… was related at some point with something that isn't there anymore :shrug: .